sources/packages/web/src/services/http/dto/Note.dto.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Base DTO for note.
+ */
+export interface NoteBaseAPIInDTO {
+  noteType: string;
+  description: string;
+}
+
+/**
+ * Notes detail DTO. This is used for view only purpose.
+ */
+export interface NoteAPIOutDTO extends NoteBaseAPIInDTO {
+  firstName: string;
+  lastName: string;
+  createdAt: Date;
+  showMore?: boolean;
```

Comment:
Is it a Vue property or an API/DTO property?

Reply:
Its a vue property

Reply:
It is the Vue property to hide and show any text more than 150 characters.

Reply:
Sorry, I was not clear enough, if it is a Vue property it should not be in the DTO.

Reply:
Thanks for doing the change @guru-aot.